src/Microsoft.DotNet.Build.Tasks/PackageFiles/CloudTest.targets

```diff
@@ -0,0 +1,522 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
OK I removed CloudTest.targets, both this one that I accidentally added and the one under CloudTestTasks.